docs/source/deploypeer/peerdeploy.md

```diff
@@ -0,0 +1,112 @@
+# Deploy the peer
+
+Before deploying a peer, make sure to digest the material in [Planning for a peer](./peerplan.html) and [Checklist for a production peer](./peerchecklist.html) which discusses all of the relevant decisions you need to make and parameters you need to configure before deploying a peer.
+
+## Download the peer binary and configuration files
+
+The Fabric peer binary and configuration files can be downloaded from [GitHub](https://github.com/hyperledger/fabric/releases) to a folder on your local system for example `fabric/`. Scroll to the Fabric release you want to download, click the **Assets** twistie, and select the binary for your system type. Extract the ZIP file and you will find all of the Fabric binaries in the `/bin` folder and the associated configuration files in the `/config` folder.
+The resulting folder structure is similar to:
+
+```
+├── fabric
+  ├── bin
+  │   ├── configtxgen
+  │   ├── configtxlator
+  │   ├── cryptogen
+  │   ├── discover
+  │   ├── idemixgen
+  │   ├── orderer
+  │   └── peer
+  └── config
+    ├── configtx.yaml
+    ├── core.yaml
+    └── orderer.yaml
+```
+
+Along with the relevant binaries, you will receive both the peer binary executable and the peer configuration file, `core.yaml`, that is required to launch a peer on the network. The other files are not required for the peer deployment but will be useful when you attempt to create or edit channels, among other tasks.
+
+**Tip:** Add the location of the peer binary to your `PATH` environment variable so that it can be picked up without fully qualifying the path to the binary executable, for example:
+
+```
+export PATH=<path to download location>/bin:$PATH
+```
+
+After you have mastered deploying and running a peer by using these two files, it is likely you will want to use the [peer image](https://hub.docker.com/r/hyperledger/fabric-peer) that Fabric publishes instead, for example in a Kubernetes or Docker deployment. For now though, the purpose of this topic is to teach you how to properly use the peer binary.
```

Comment:
Suggestion for positioning this content better:

After you have mastered deploying and running a peer by using these two files, it is likely you will want to use the [peer image](https://hub.docker.com/r/hyperledger/fabric-peer) that Fabric publishes instead, for example in a Kubernetes or Docker deployment. For now though, the purpose of this topic is to teach you how to properly use the peer binary so you can take that knowledge and apply it to the production environment of your choice.